crates/common/rlp/encode.rs

```diff
@@ -13,6 +12,49 @@ pub fn encode<T: RLPEncode>(value: T) -> Vec<u8> {
     buf
 }
 
+/// Computes the length needed for a given payload length
+#[inline]
+pub const fn list_length(payload_len: usize) -> usize {
+    if payload_len < 56 {
+        // short prefix
+        1 + payload_len
+    } else {
+        let be = payload_len.to_be_bytes();
+        let mut i = 0;
+        while i < be.len() && be[i] == 0 {
+            i += 1;
+        }
+        let be_len = be.len() - i;
+        1 + be_len + payload_len
+    }
+}
+
+/// Computes the length needed for a given bytes length and first byte
+#[inline]
+pub const fn bytes_length(bytes_len: usize, first_byte: u8) -> usize {
+    if bytes_len == 0 {
+        return 1;
+    }
```

Comment:
This isn't needed, since we also receive the first byte, so we're assuming `bytes_len` isn't 0.